deploy/role.yaml

```diff
@@ -1,3 +1,5 @@
+# Not currently used, but will come in handy when we create CSV file
+#   Delete after CSV absorbs rbac rules
```

Comment:
We'll probably have to write this stuff in Go...but you probably need Cluster level permissions to create things in different namespaces.

Reply:
Maybe we can do something with role aggregation here.  I think we only need Admin.